### PR TITLE
website/integrations: Addition of phpIPAM SAML integration documentation

### PR DESCRIPTION
**Pull request by aaronnad**
## Changes
* Adds documentation to support additional infrastructure services for remote auth
* A search in github shows there were no inital issues for this, but this had me stumped for a few days and figured it would help future users get setup quickly.
* No breaking changes

## Additional
Added in screenshots to help users visualise the changes they are making and rough expected outcomes of what to expect.

### CODE REVIEW

**Comment by aaronnad** [comment]
```
➜  phpipam git:(website/integrations/services/phpipam) ✗ npm run prettier-check                                                                                                                                                    

> @goauthentik/website@0.0.0 prettier-check
> prettier --check .

Checking formatting...
All matched files use Prettier code style!
```

Ran the prettier-check locally and it now passes, so the next bout of builds should successfully complete all the way - last time it failed on the ci-website-mark on the prettier build

Screenshot 2023-04-11 194933

**Comment by aaronnad** [comment]
Resolved the codespell linting typo of `website/integrations/services/phpipam/index.md:203: certifacte ==> certificate`

File: website/integrations/services/phpipam/index.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    ```
+
+4. groups
+
+    - Name: phpipam-groups
+    - SAML Attribute Name: groups
+    - Expression:
+
+    ```python
+    if  ak_is_group_member(request.user, name="operator-permission-group"):
```
Comment (BeryJu) [inline website/integrations/services/phpipam/index.md:128]:
```suggestion
    if ak_is_group_member(request.user, name="operator-permission-group"):
```

File: website/integrations/services/phpipam/index.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        return "Operators"
+    elif ak_is_group_member(request.user, name="guest-permission-group"):
+        return "Guests"
+    ```
+
+5. modules
+    - Name: phpipam-modules
+    - SAML Attribute Name: groups
```
Comment (BeryJu) [inline website/integrations/services/phpipam/index.md:136]: I'm not sure if having two mappings with the same SAML attribute name is correct, authentik should handle it correctly (two XML attribute objects will be created but both will have the same name), but I'm not sure if the application will understand that correctly

File: website/integrations/services/phpipam/index.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    - Expression:
+    ```python
+    if  ak_is_group_member(request.user, name="operator-permission-group"):
```
Comment (BeryJu) [inline website/integrations/services/phpipam/index.md:139]:
```suggestion
    if ak_is_group_member(request.user, name="operator-permission-group"):
```

File: website/integrations/services/phpipam/index.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+        return "*:2"
+    elif ak_is_group_member(request.user, name="guest-permission-group"):
+        return "*:1"
+    ```
+
+![](./phpipam-property-mappings.png)
+
+### Step 3 - Provider creation
+
+-   Select Create => SAML Provider
```
Comment (BeryJu) [inline website/integrations/services/phpipam/index.md:149]:
```suggestion
-   Select Create -> SAML Provider
```

File: website/integrations/services/phpipam/index.md (hunk 1 above)
Comment (aaronnad) [reply website/integrations/services/phpipam/index.md:128]: Fixed this as well now. Extra space was also in my provider config, so fixed that locally too! Thanks for the spot.

File: website/integrations/services/phpipam/index.md (hunk 2 above)
Comment (aaronnad) [reply website/integrations/services/phpipam/index.md:136]: You are indeed correct. That should be `SAML Attribute Name: modules`.

File: website/integrations/services/phpipam/index.md (hunk 4 above)
Comment (aaronnad) [reply website/integrations/services/phpipam/index.md:149]: All of my '->' are '=>'. Updated these to now be single `->` instead.

**Comment by BeryJu** [comment]
Yeah don't worry about that job failing, that's due to us upgrading to Django 4.2 since the latest stable release and that job is a bit finicky

**Review by tanberry — APPROVED** [review]
Approve based on @BeryJu review so that we can merge

**Comment by tanberry** [comment]
Thanks @aaronnad for the PR!!

**Comment by aaronnad** [comment]
Thank you for the thorough review! Hoping to add more in the future as I build upon some of my services with Authentik

**Pull request merged**